cmd/eve.go

```diff
@@ -177,6 +177,11 @@ func newConsoleEveCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
 		Short: "telnet into eve",
 		Long:  `Telnet into eve.`,
 		Run: func(cmd *cobra.Command, args []string) {
+			// do we have any ideas how to provide defaults?
+			// ot should we set Host in config as well as an option
```

Comment:
We have merge and cfg from CLI, where we can add default values. But I think would be nice to merge default values and config generation from template by just having one instance of config file with parameters :)